rca/landingpages/migrations/0002_rename_model.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.9 on 2020-04-14 09:53
```

Comment:
Could the migrations be consolidated easily? (not break anyone elses build if deleted and recreated as one)

Reply:
I wanted to do this but it's already gone to staging so I'm leaving it as is